styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
```

Comment:
I would leave the `max-width` off right now since there isn't really any strong evidence to suggest your users will be negatively affected by an unlimited width.

In a future release, you can set a `max-width` and center the content as an enhancement.